fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/storage/Storage.java

```diff
@@ -165,9 +165,9 @@ default long simulateExtract(T resource, long maxAmount, @Nullable TransactionCo
 	 * @return An iterable over the contents of this storage.
 	 * @see #iterator
 	 */
-	@SuppressWarnings("unchecked")
-	default Iterable<StorageView<T>> iterable(TransactionContext transaction) {
-		return () -> (Iterator<StorageView<T>>) iterator(transaction);
+	@SuppressWarnings({"rawtypes", "unchecked"})
+	default Iterable<? extends StorageView<T>> iterable(TransactionContext transaction) {
+		return () -> (Iterator) iterator(transaction);
```

Comment:
Do you need this cast at all?

Reply:
IntelliJ says that I need it
![image](https://user-images.githubusercontent.com/13494793/159177505-06ff20d0-4d7d-4cf5-8f82-4fe02fa07944.png)

I suspect that it's because the wildcard type cannot be instantiated directly.